src/main/java/io/lettuce/core/protocol/CommandHandler.java

```diff
@@ -478,6 +482,13 @@ private void attachTracing(ChannelHandlerContext ctx, RedisCommand<?, ?, ?> comm
             Tracer.Span span = tracer.nextSpan(context);
             span.name(command.getType().name());
 
+            String redisUriStr = ctx.channel().attr(ConnectionBuilder.REDIS_URI).get();
```

Comment:
I see that there is a chance that the attribute is missing, so it would be good to make this whole logic conditional, e.g.

```java
        if (channel.hasAttr(ConnectionBuilder.REDIS_URI)) {
           ...
        }
```

Reply:
It makes sense, I updated 🙇

Reply:
Thanks!